scripts/linters/js_ts_linter.py

```diff
@@ -1014,6 +1014,142 @@ def _check_comments(self):
             summary_messages.append(summary_message)
         return summary_messages
 
+    def _check_oppia_angular_root(self):
+        """Finds all @Injectable classes and makes sure that they are added to
+            Oppia root and Oppia Angular Root.
+
+        Returns:
+            all_messages: str. All the messages returned by the lint checks.
+        """
+        def get_injectable_class_name(file_content):
+            """Extarcts the class name from a file that has an Injectable
+                class.
+
+            Args:
+                file_content: str. File content of the file that has an
+                    Injectable class.
+
+            Returns:
+                tuple(str, str). A two-tuple of class name and class name in
+                camelCase.
+            """
+
+            # Going through all ts files and looking for @Injectable classes.
+            class_name = file_content.split(
+                '@Injectable({')[1].split(
+                    'export class ')[1].split('{')[0].replace(' ', '')
+            camel_case_class_name = class_name[0].lower() + class_name[1:]
+            return class_name, camel_case_class_name
+
+        #pylint: disable=invalid-name
+        OPPIA_ANGULAR_ROOT_PATH = (
+            './core/templates/components/oppia-angular-root.component.ts')
+        #pylint: disable=invalid-name
+        OPPIA_ROOT_DIRECTIVE_PATH = (
```

Comment:
Remove pylint params and make these variables local i.e. change them to lowercase.

Reply:
Done!